gammapy/modeling/tests/test_fit.py

```diff
@@ -319,3 +322,41 @@ def test_stat_contour():
 
     # Check that original value state wasn't changed
     assert_allclose(dataset.models.parameters["y"].value, 300)
+
+
+def test_with_prior():
```

Comment:
This test is certainly at the wrong position, but I added it to show how setting and evaluating priors can work.